.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bicep-types"]
+	path = bicep-types
+	url = https://github.com/Azure/bicep-types
```

Comment:
Is this something we need long-term? Adding a submodule means we have to update a lot of instructions.

Do you think there's a way we could decouple the repos so we don't need the bicep generate in the Radius repo?

Reply:
Sure, we could remove the `hack` folder into its own repository similar to how the `bicep-types-aws` repo is. So that repository would host the submodule instead of the main radius repo and we'd handle uploads to ACR through the new repo. 

The submodule should just require a `git submodule init && git submodule update` if the user wants to use it locally. I don't think it affects development unless someone is working with the bicep generator